package.json

```diff
@@ -23,7 +23,8 @@
         "examples/3-apps/1-full-stack",
         "examples/3-apps/2-microservices",
         "examples/4-integrations/1-react",
-        "examples/4-integrations/2-vue"
+        "examples/4-integrations/2-vue",
+        "examples/4-integrations/3-svelte"
```

Comment:
The Svelte example is missing in the `examples/README.md` file.

Reply:
Added to the read me